tests/e2e/config/mainnet.json

```diff
@@ -0,0 +1,8 @@
+{
+    "baseUrl": "https://explorer.ec.stage.rnd.land/",
```

Comment:
You must decide whether to have `/` at the end or not because in `local.json` you don't have it. It might result in some unexpected behavior. Personally, I'm a fan of not having `/` in baseUrl but it's up to you. Be consistent, that's important.